projects/content-lib/src/lib/content-lib.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+@Injectable()
+export class ContentLibService {
+
+    constructor(private _httpClient: HttpClient, private _router: Router) {
+    }
+
+    getArticle(apiBaseUrl, article_alias) {
+        const url = apiBaseUrl + 'index.php?option=com_api&app=articles&resource=article&format=raw&article_alias=' + article_alias;
```

Comment:
Move this to the top of the class, not in this method. 
The base URL should be injected into this component via the module config cc @sudhir-j-sapkal @Pratikshakhandagale